docker/buildimage/install-go.sh

```diff
@@ -6,7 +6,7 @@ GOPATH="/go"
 
 export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
 
-mkdir -p "$GOPATH/src" "$GOPATH/bin"
+mkdir -p "$GOPATH/src" "$GOPATH/bin" "$GOPATH/scripts"
```

Comment:
I’m curious why this is needed in the general build image we use for all the go apps. Does a build fail if it is missing?

Reply:
yes

Reply:
scripts/ds3cp=/opt/bendo/scripts/ds3cp
--
75 | /bin/sh: scripts/ds3cp=/opt/bendo/scripts/ds3cp: No such file or directory
76 | make: *** [rpm] Error 127